test/unit-test.js

```diff
@@ -0,0 +1,46 @@
+var expect = require('chai').expect;
+var assert = require('chai').assert;
+var should = require('chai').should;
+
+const fs = require('fs');
+const h5 = require('../index')
+const chai = require('chai')
```

Comment:
It looks like we're only using `assert`, which you already defined above.

I'd probably do something like `const { assert } = require('chai');`, but I'm a hipster so whatever.